tardis/model/base.py

```diff
@@ -204,15 +142,17 @@ def dilution_factor(self, value):
 
     @property
     def t_radiative(self):
-        return self.radiation_field.t_radiative
+        return self.radiation_field_state.t_radiative[
+            self.geometry.v_inner_boundary_index : self.geometry.v_outer_boundary_index
+        ]
 
     @t_radiative.setter
-    def t_radiative(self, value):
-        if len(value) == self.no_of_shells:
-            self.radiation_field.t_radiative = value
+    def t_radiative(self, new_t_radiative):
+        if len(new_t_radiative) == self.no_of_shells:
+            self.radiation_field_state.t_radiative = new_t_radiative
         else:
             raise ValueError(
-                "Trying to set t_radiative for unmatching number of shells."
+                "Trying to set t_radiative for unmatching number" "of shells."
```

Comment:
Should there be a number here? The `" "` doesn't seem necessary. I would also replace "unmatching" with "different" or "unexpected"